samples/nrf9160/azure_fota/README.rst

```diff
@@ -0,0 +1,161 @@
+.. _azure_fota_sample:
+
+nRF9160: Azure FOTA
+#################
+
+The Azure firmware over-the-air (Azure FOTA) sample shows how to perform an over-the-air firmware update of an nRF9160 device using Azure IoT Hub and Azure FOTA libraries.
+
+
+Overview
+*********
+
+The sample connects to the configured `Azure IoT Hub`_ and waits for incoming device twin updates containing firmware update information.
+By default, the device twin document is requested upon connecting to the IoT Hub, so any existing firmware information in the ``desired`` property will be parsed.
+See :ref:`lib_azure_fota` for more details on the content of the firmware information in the device twin.
+
+.. note::
+   A file server instance that hosts the new firmware image is required for this sample.
+   The Azure FOTA library does not require a specific host, but has been tested using Azure Storage that shares the same root certificate the Azure IoT Hub.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. include:: /includes/boardname_tables/sample_boardnames.txt
+   :start-after: set6_start
+   :end-before: set6_end
+
+The sample requires an Azure IoT Hub, and optionally an Azure IoT Hub Device Provisioning Service instance if the device is not already registered with the IoT hub.
+
+.. include:: /includes/spm.txt
+
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/nrf9160/azure_fota`
+
+.. include:: /includes/build_and_run_nrf9160.txt
+
+.. _configuring:
+
+Sample configuration
+====================
+
+Before you build the sample, check and update the following configuration options:
+
+.. option:: AZURE_FOTA_APP_VERSION - Application version
+
+   The version string is printed when the sample starts.
+   Use this information to verify that the FOTA update worked.
+
+.. option:: AZURE_FOTA_TLS - Use HTTPS for downloads.
+
+   By default TLS is enabled for downloads.
+   Currently, the transport protocol must be configured at compile-time.
+
+.. option:: AZURE_FOTA_SEC_TAG - Security tag for TLS credentials
+
+   If HTTPS is used as transport layer, certificates must be provisioned to the device and the sec tag option must be set accordingly.
+
+.. option:: AZURE_IOT_HUB_NAME - Azure IoT Hub name.
```

Comment:
The IoT Hub name was used in the early stages of development instead of the full hostname. I've moved to using the full hostname name and removed remnants of the IoT Hub name usage.